src/App.js

```diff
@@ -89,31 +102,32 @@ class Board extends React.Component {
             }
             );
         }
-        if ( key === 'Backspace' ) {
-            if ( currentWordLen > 0 ) {
-                this.setState(( state ) => {
+        if (key === 'Backspace') {
+            if (currentWordLen > 0) {
+                this.setState((state) => {
                     const { currentAttempt, attempts } = state;
-                    const newWord = attempts[currentAttempt].slice( 0, currentWordLen-1 );
+                    const newWord = attempts[currentAttempt].slice(0, currentWordLen - 1);
                     const newAttempts = attempts;
                     newAttempts[currentAttempt] = newWord;
                     return { attempts: newAttempts };
-                } );
+                });
             }
         }
     };
 
     renderBoard = () => {
         const { attempts, currentAttempt } = this.state;
-        return attempts.map(( word, index ) =>
+        this.loadFromLocalStorage();
```

Comment:
I don't think it is a good idea to read from local storage on each render. When the state changes it causes re-renders to occur. Since `loadFromLocalStorage ` doesn't interact with the state it currently doesn't do anything but if we made it interact with the state it would cause re-renders to occur as well as potentially losing input from the user